recipes/cpprate/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "cpprate" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/tmaklin/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 9734f5e9da27404ad12e5d426d0547d0b35e91b86b37f9f34367570fb5660af4
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake >=3.1
+    - make
+    - llvm-openmp        # [osx]
+    - libgomp            # [linux64]
+  host:
+    - libblas
+    - eigen
+    - bzip2
+    - xz
+  run:
+    - bzip2
+    - xz
+
+test:
+  commands:
+    - cpprate --help 2>&1 | grep Usage
+
+about:
+  home: https://github.com/tmaklin/cpprate
+  summary: 'Variable Selection in Black Box Methods with RelATive cEntrality (RATE) Measures'
+  description: |
+    cpprate is a reimplementation of https://github.com/lorinanthony/rate in C++.
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file:
+    - LICENSE
+    - external/bs_thread_pool/LICENSE.txt
```

Comment:
Please include the licenses for the external header only libraries that CMake is fetching during the build process! https://github.com/tmaklin/cpprate/tree/main/config

Dynamically link if any of these are not header-only.